src/_gettsim/transfers/arbeitsl_geld_2/sgb_ii_bedarf.py

```diff
@@ -0,0 +1,275 @@
+    elif alter <= anteile[4]["max_alter"] and gleiche_fg_kindergeldempfänger_kind:
+        out = regelsatz * anteile[4]["anteil"]
+    else:
+        out = 0.0
+
+    return float(out)
+
+
+@policy_info(start_date="2011-01-01", name_in_dag="arbeitsl_geld_2_kindersatz_m")
+def arbeitsl_geld_2_kindersatz_m_ab_2011(
+    alter: int,
+    gleiche_fg_kindergeldempfänger_kind: bool,
+    arbeitsl_geld_2_params: dict,
+) -> float:
+    """Basic monthly subsistence / SGB II needs of children since 2011.
+
+    Note: Since 2023, Arbeitslosengeld 2 is referred to as Bürgergeld.
+
+    Parameters
+    ----------
+    alter
+        See basic input variable :ref:`alter`.
+    gleiche_fg_kindergeldempfänger_kind
+        See :func:`gleiche_fg_kindergeldempfänger_kind`.
+    arbeitsl_geld_2_params
+        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
+
+    Returns
+    -------
+    SGB II needs of child
+
+    """
+
+    kindersofortzuschl = arbeitsl_geld_2_params.get("kindersofortzuschl", 0.0)
+
+    if (
+        alter <= arbeitsl_geld_2_params["regelsatz"][6]["max_alter"]
+        and gleiche_fg_kindergeldempfänger_kind
+    ):
+        out = arbeitsl_geld_2_params["regelsatz"][6]["bedarf"] + kindersofortzuschl
+    elif (
+        alter <= arbeitsl_geld_2_params["regelsatz"][5]["max_alter"]
+        and gleiche_fg_kindergeldempfänger_kind
+    ):
+        out = arbeitsl_geld_2_params["regelsatz"][5]["bedarf"] + kindersofortzuschl
+    elif (
+        alter <= arbeitsl_geld_2_params["regelsatz"][4]["max_alter"]
+        and gleiche_fg_kindergeldempfänger_kind
+    ):
+        out = arbeitsl_geld_2_params["regelsatz"][4]["bedarf"] + kindersofortzuschl
+    elif gleiche_fg_kindergeldempfänger_kind:  # adult children with parents in FG
+        out = arbeitsl_geld_2_params["regelsatz"][3] + kindersofortzuschl
+    else:
+        out = 0.0
+
+    return float(out)
+
+
+@policy_info(end_date="2010-12-31", name_in_dag="arbeitsl_geld_2_erwachsenensatz_m")
+def arbeitsl_geld_2_erwachsenensatz_bis_2010_m(
+    _arbeitsl_geld_2_alleinerz_mehrbedarf_m: float,
+    arbeitsl_geld_2_kindersatz_m: float,
+    p_id_einstandspartner: int,
+    arbeitsl_geld_2_params: dict,
+) -> float:
+    """Basic monthly subsistence / SGB II needs for adults without dwelling.
+
+    Parameters
+    ----------
+    _arbeitsl_geld_2_alleinerz_mehrbedarf_m
+        See :func:`_arbeitsl_geld_2_alleinerz_mehrbedarf_m`.
+    arbeitsl_geld_2_kindersatz_m
+        See :func:`arbeitsl_geld_2_kindersatz_m`.
+    p_id_einstandspartner
+        See basic input variable :ref:`p_id_einstandspartner`.
+    arbeitsl_geld_2_params
+        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
+
+    Returns
+    -------
+
+    """
+    if p_id_einstandspartner > 0:  # BG with 2 adults
+        out = arbeitsl_geld_2_params["regelsatz"] * (
+            arbeitsl_geld_2_params["anteil_regelsatz_erwachsene"]["zwei_erwachsene"]
```

Comment:
Do I gather correctly that this gives the sum where it should give the average?

We should make a note that we want to return the average in order to have both adults symmetric rather than keeping the difference between RBS 1 and 2 around.

Reply:
As discussed on the phone, the Regelsatz is on individual level already in the parameter files (as it is in the law). So the result here is correct.